src/pymor/models/iosys.py

```diff
@@ -505,6 +575,44 @@ def to_abcde_files(self, files_basename):
         if E is not None:
             _mmwrite(Path(files_basename + '.E'), E)
 
+    def _compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
+                 solution_error_estimate=False, output_error_estimate=False,
+                 output_d_mu_return_array=False, mu=None, **kwargs):
+        if not solution and not output:
+            return {}
+
+        # solution computation
+        mu = mu.with_(t=0.)
+        X0 = self.initial_data.as_range_array(mu)
+        input_op = NumpyGenericOperator(
+            lambda U, mu: U * mu['input'],
+            dim_range=self.dim_input,
+            parameters={'input': self.dim_input},
+            linear=True,
+        )
```

Comment:
I think the uninitiated would not be able to come up with this solution. So maybe we should have an `InputOperator` with this functionality?